src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -855,8 +869,24 @@ private void SendGraphicsToView(GraphicsUpdateParams parameters)
             PerVertexMesh = parameters.PerVertexMesh;
             Text = parameters.Text;
 
+            linesView.Attach(watch_view.RenderHost);
+            linesSelectedView.Attach(watch_view.RenderHost);
+            pointsView.Attach(watch_view.RenderHost);
+            meshView.Attach(watch_view.RenderHost);
+            meshSelectedView.Attach(watch_view.RenderHost);
+            perVertexMeshView.Attach(watch_view.RenderHost);
+            textView.Attach(watch_view.RenderHost);
+
             // Send property changed notifications for everything
             NotifyPropertyChanged(string.Empty);
+
+            Points = null;
```

Comment:
We can null these collections here as the geometry has been handed off to the GPU. We don't need these collections any more. Again, in the new world, we'll need to see whether we can do this. I'm not sure that we can if we need to have the collections available for selection change.